AdvLoggerPkg/AdvLoggerPkg.ci.yaml

```diff
@@ -77,5 +77,12 @@
           POSTMEM,
           MMARM
         ]
+    },
+
+    "RustHostUnitTestPlugin": {
+        "Coverage": .75,
+        "CoverageOverrides": {
+            "RustAdvancedLoggerDxe": 0.0
+        }
     }
 }
```

Comment:
I'll add a newline to this file. Want to wait for the current PR builds to finish before pushing again.

Reply:
Done